packages/kotti-ui/source/kotti-breadcrumb/KtBreadcrumb.vue

```diff
@@ -25,46 +25,76 @@
 	</div>
 </template>
 
-<script>
-export default {
+<script lang="ts">
+import { Yoco } from '@3yourmind/yoco'
+import { defineComponent } from '@vue/composition-api'
+import { isBoolean, isString } from 'lodash'
+
+import { isYocoIcon } from '../validators'
+
+import { KottiBreadcrumb } from './types'
+
+const breadcrumbValidator = (
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	value: any,
+): value is KottiBreadcrumb.Breadcrumb =>
+	value !== null &&
+	typeof value === 'object' &&
+	isBoolean(value.isCompleted) &&
+	isString(value.title) &&
+	isString(value.to)
+
+export default defineComponent<KottiBreadcrumb.PropsInternal>({
 	name: 'KtBreadcrumb',
 	props: {
-		breadcrumbs: { type: Array, required: true },
+		breadcrumbs: {
+			type: Array,
+			required: true,
+			validator: (value: unknown) =>
+				Array.isArray(value) && value.every(breadcrumbValidator),
+		},
 		separator: {
 			type: Object,
 			default() {
```

Comment:
Should use arrow function & type annotate the result. Otherwise, TS will not check if you keep the default up-to-date in future iterations on `KtBreadcrumb`